Blockzilla/OverlayView.swift

```diff
@@ -15,19 +15,40 @@ protocol OverlayViewDelegate: class {
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-    private var searchButtonGroup = [InsetButton]()
-    private let searchSuggestionsCount = 4 // Five search buttons in total since indexing starts from 0.
-    private var presented = false
-    private var searchQuery = ""
     private let copyButton = UIButton()
     private let findInPageButton = InsetButton()
+    private let searchSuggestionsPrompt = SearchSuggestionsPromptView()
+    private let searchButton = InsetButton()
+    private var searchButtonGroup = [InsetButton]()
+    private let searchSuggestionsCount = 4 // Five search buttons in total since indexing starts from 0.
     private let topBorder = UIView()
     public var currentURL = ""
-
+    private var presented = false
+    private var searchQuery = ""
```

Comment:
See above. If there's an actual logical reason to move, feel free to comment!

Reply:
Nope! Was just organizing it by grouping by lets and vars and maintaining alphabetical order but no actual logical reason! Will restore as before :)